### ISSUE TITLE
Init called twice ver1.4.3

### ISSUE THREAD

**Issue by livarkhal** [gw2-addon-loader/GW2-Addon-Manager#157]
Hi,

I tried you addon manager today (i download the v1.4.3) and it not work for me ...
I have this in logs : 

[INFO] core | Addon loader v1.0 (v1.1 r33) initialized
[INFO] core | Loading addons from "./addons" path
[INFO] core | Loading arcdps
[INFO] core | Loaded d3d9_wrapper v2.0 r21
[INFO] core | Loaded arcdps v1.0 r403
[INFO] core | Loading d3d9_wrapper
[INFO] core | Loading gw2radial
[INFO] core | Loaded gw2radial v2.1 r1
[INFO] core | Done loading addons
[INFO] d3d9_wrapper | Loading system dxgi
[WARN] core | Init called twice without proper unload. If your addon is not working, make sure you handle this situation!
[INFO] d3d9_wrapper | Loading system d3d11
[WARN] core | Init called twice without proper unload. If your addon is not working, make sure you handle this situation!
[INFO] d3d9_wrapper | Loading system dxgi
[WARN] core | Init called twice without proper unload. If your addon is not working, make sure you handle this situation!
[INFO] d3d9_wrapper | Loading system d3d11

Ty for help :)

**Comment by Friendly0Fire** [comment]
That indicates everything's working fine. What is not working?

**Comment by livarkhal** [comment]
Oh sorry ... When i launch the game, i don't see arcdps or gw2radial, and when i push Shift+Alt+H for arcdps options or Shift+Alt+M for gw2radial options, nothing happens ...

**Comment by Friendly0Fire** [comment]
Try removing arcdps and see if Radial works and vice-versa.

**Comment by livarkhal** [comment]
Ok, radial works alone, not arcdps.
And the delete option in your app doesn't seem to work ... i need to "Reset to clean install" to uninstall addons.

**Comment by WizardsCreation** [comment]
I can't get radial to work at all. Even by itself and or a manual install.

**Comment by SleepyPwasnotavailable** [comment]
> 
> 
> Oh sorry ... When i launch the game, i don't see arcdps or gw2radial, and when i push Shift+Alt+H for arcdps options or Shift+Alt+M for gw2radial options, nothing happens ...

Same for me

**Comment by KevinExtremo** [comment]
I am experiencing the same issue. Wont work. Manual install I can get to work but only to such extent that I go ingame and the radial menu is invisible. I really don't understand what is going on. Running on gx9, even with the command line argument.

I tried to install the loader-core myself, no luck. I installed radial myself, no luck. Made sure I have the latest c++ redistributables (2015, 2017, 2019). No luck.

With the addon manager and launch buddy I managed to pinpoint that it seems to crash when the d3d9.dll and the d3d9_wrapper.dll are loaded at the same time. There seems to be some conflict leading to a crash.
